### DISCUSSION TITLE
Change the start day of the week, to reflect Sprints starting on Wednesdays?

### DISCUSSION THREAD

**Discussion by jenniferrosemitchell21** [nbrown02/FlowViz#109] [Q&A]
Hi Nick, Is there any way to change the start day of the week? i.e. to Wednesday to mirror the Sprints

**Comment by nbrown02** [comment]
Yes you can :)

There's a team I work with who want the metrics to have a week start date
of every Thursday, so I rename the original WeekStartingDate to
WeekStartingDate2 and use this formula:
WeekStartingDate = IF( Dates[DayOfWeek] <= 5, Dates[WeekStartingDate2] - 3,
Dates[WeekStartingDate2] + 4)
